exporter/datadogexporter/translate_traces.go

```diff
@@ -291,17 +296,35 @@ func aggregateSpanTags(span pdata.Span, datadogTags map[string]string) map[strin
 	// predefine capacity as at most the size attributes and global tags
 	// there may be overlap between the two.
 	spanTags := make(map[string]string, span.Attributes().Len()+len(datadogTags))
+
 	for key, val := range datadogTags {
 		spanTags[key] = val
 	}
 
 	span.Attributes().ForEach(func(k string, v pdata.AttributeValue) {
 		spanTags[k] = tracetranslator.AttributeValueToString(v, false)
+
 	})
 
+	spanTags[tagContainersTags] = buildDatadogContainerTags(spanTags)
 	return spanTags
 }
 
+// buildDatadogContainerTags returns container and orchestrator tags belonging to containerID
+// as a comma delimeted list for datadog's special container tag key
+func buildDatadogContainerTags(spanTags map[string]string) string {
+	var b strings.Builder
+
+	if val, ok := spanTags[conventions.AttributeContainerID]; ok {
+		b.WriteString(fmt.Sprintf("%s:%s,", "container_id", val))
+	}
+	if val, ok := spanTags[conventions.AttributeK8sPod]; ok {
+		b.WriteString(fmt.Sprintf("%s:%s,", "pod_name", val))
+	}
+
+	return b.String()
+}
```

Comment:
Nit: with this logic, as long as one of the attributes is found, the returned value will end with an unneeded comma (eg. `container_id:3249847017410247,pod_name:example-pod-name,`). Could we remove the final comma in that case? I don't think the final comma breaks anything, but it'd be cleaner to not have it.